modules/guides/pages/create-user-defined-function.adoc

```diff
@@ -60,18 +60,18 @@ From this list select the library containing your function.
 REST API::
 +
 --
-Run a `curl` command from the shell to add a N1QL User-Defined Function that calls your Javascript function.
+Run a `curl` command from the shell to add an {sqlpp} User-Defined Function that calls your Javascript function.
```

Comment:
As above, I think we should leave the indefinite article unchanged; it's one less thing to worry about.